include/dump_offload.hpp

```diff
@@ -453,11 +454,11 @@ inline void requestRoutes(App& app)
             conn.close();
             return;
         }
-
         std::string dumpEntry =
             url.substr(pos1 + startDelimiter.length(),
                        pos2 - pos1 - startDelimiter.length());
-
+        BMCWEB_LOG_CRITICAL << "INFO:Request open for system dump offload with"
```

Comment:
This can be removed as the url contains the dump entry number

Reply:
Done